shared/actions/teams/index.js

```diff
@@ -387,6 +389,7 @@ const _getDetails = function*(action: Constants.GetDetails): Saga.SagaGenerator<
     ])
   } finally {
     yield Saga.put(replaceEntity(['teams', 'teamNameToLoading'], I.Map([[teamname, false]])))
+    yield Saga.put(createDecrementWaiting(waitingKey))
```

Comment:
is this a `TODO replace teamNameToLoading`?

Reply:
Yep, we should talk about it! Added a TODO comment.